src/Symfony/Component/Notifier/Bridge/Discord/DiscordTransport.php

```diff
@@ -59,7 +60,7 @@ public function supports(MessageInterface $message): bool
     protected function doSend(MessageInterface $message): SentMessage
     {
         if (!$message instanceof ChatMessage) {
-            throw new LogicException(sprintf('The "%s" transport only supports instances of "%s" (instance of "%s" given).', __CLASS__, ChatMessage::class, get_debug_type($message)));
+            throw new UnsupportedMessageTypeException(__CLASS__, ChatMessage::class, $message);
```

Comment:
This requires `"symfony/notifier": "^5.3"` in composer.json 
Same everywhere in this PR and in open PRs about new notifiers ;-)

Reply:
Good catch 👍🏻

Reply:
Done